app/Resources/views/search/events.html.twig

```diff
@@ -0,0 +1,17 @@
+{% extends 'search/layout.html.twig' %}
+
+{% block results %}
+  {% for result in results %}
+    <a href="#" style="float: left; width: 200px; background-color: lightgrey; margin: 10px; display: block">
+      <h3>The event name</h3>
+      <h4>
+        <small>
+          {{ '1 participant|%count% participants'|transchoice(3) }}
+          {% if 1 %}
+            &bullet; Angoulème
```

Comment:
`CommitteeEvent` was not in master at the opening of this PR. Now I can add this.